debian/rules

```diff
@@ -18,7 +18,8 @@ else
 endif
 
 extraopts += -DWITH_OCF=ON -DWITH_LTTNG=ON
-extraopts += -DWITH_PYTHON3=ON -DWITH_MGR_DASHBOARD_FRONTEND=OFF
+extraopts += -DWITH_MGR_DASHBOARD_FRONTEND=OFF -DWITH_MGR_PYTHON3=ON
+extraopts += -DWITH_PYTHON3=ON -DWITH_PYTHON2=OFF -DMGR_PYTHON_VERSION=3
```

Comment:
@alfredodeza could you reconsider https://github.com/ceph/ceph/pull/25969#discussion_r336583632 ?

Reply:
I thought this change was good enough. Would you prefer to have everything python3 related in one line? My take on it was that it was becoming too long.